tests/test_FindTest.py

```diff
@@ -176,8 +186,8 @@ def test_find_citations(self):
             ('2006-Ohio-2095',
              [case_citation(0, source_text='2006-Ohio-2095', page='2095',
                             reporter='Ohio', volume='2006')]),
-            ('2017 IL App (4th) 160407WC',
-             [case_citation(0, page='160407WC', reporter='IL App (4th)',
+            ('2017 IL App (4th) 160407',
+             [case_citation(0, page='160407', reporter='IL App (4th)',
```

Comment:
So I get that this is changed because we no longer allow text that abuts a citation, but is `160407WC` actually a valid page in `IL App`? I have no idea, but if so, is that something that we would put in `reporters_db` now? (I haven't been following too closely what's been going on over there.)

Reply:
Yeah, this is handled in reporters-db [now](https://github.com/freelawproject/reporters-db/pull/47) and should work -- it's just not on PyPI yet.